src/test/java/com/glovoapp/planningpoker/e2e/Clear_ShowVotes_ShareSessionLink/ClearButtonTest.java

```diff
@@ -0,0 +1,41 @@
+package com.glovoapp.planningpoker.e2e.Clear_ShowVotes_ShareSessionLink;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class ClearButtonTest {
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private SessionPage sessionPage = new SessionPage(driver);
+    final String username = "michal";
+    final String ticketname = "ticket";
```

Comment:
Please use `camelCase` for variable names.
Also, this is a constant and should be `static`:

```suggestion
    private static final String TICKET_NAME = "ticket";
```